samples/boards/96b_argonkey/sensors/prj.conf

```diff
@@ -6,9 +6,5 @@ CONFIG_LED=y
 CONFIG_GPIO=y
 CONFIG_SENSOR=y
 CONFIG_SENSOR_LOG_LEVEL_DBG=y
-CONFIG_HTS221=y
-CONFIG_LPS22HB=y
-CONFIG_VL53L0X=y
-CONFIG_LSM6DSL=y
 CONFIG_LP3943=y
```

Comment:
What about that one?

Reply:
Its not a sensor, its a LED and hasn't been converted over to the new Kconfig logic/style.

Reply:
Ah right, I've looked too briefly :)